recipes/gubbins/meta.yaml

```diff
@@ -28,28 +28,27 @@ requirements:
   host:
     - perl
     - zlib
-    - python
-    - scipy >=1.5.3
+    - python >=3.8
+    - scipy
     - setuptools
-    - dendropy  >=4.0.2
-    - biopython >=1.59
-    - numpy <=1.23.0
-    - numba >=0.53
+    - dendropy
+    - biopython
+    - numpy
+    - numba
   run:
-    - python
-    - scipy >=1.5.3
-    - reportlab >=3.0
-    - dendropy  >=4.0.2
-    - biopython >=1.59
-    - multiprocess >=0.70
+    - python >=3.8
+    - scipy
+    - dendropy
+    - biopython
+    - multiprocess
     - setuptools
     - raxml =8.2.12
     - iqtree >=2.2
     - rapidnj
     - raxml-ng =1.0.1
     - fasttree =2.1.10
-    - numpy >=1.2
-    - numba >=0.56
+    - numpy <=1.23.0
```

Comment:
you dropped the numpy version restriction above for `host:` so is it needed?

Reply:
Well spotted, thanks - I will specify the version for `host` as well for the time being, as that will hopefully consistently avoid the problematic numpy versions, but hopefully we will soon be able to provide a >= constraint.